Astral Pathfinder/UI/PlanetInfo.cpp

```diff
@@ -55,66 +81,141 @@ void PlanetInfo::clean() {
 void PlanetInfo::setUiTextures(Planet p) {
   setBoxes(p);
   
-  if (slider.isInitialized()) {
-    // Sets correct slider position for given planet
-    float ratio = p.getFertilityPercent()/100.0f;
-    SDL_Rect tmpR = slider.getBaseRect();
-    slider.setSliderPosition(static_cast<int>(tmpR.w * ratio));
+  if (!sliderOne.isInitialized() && !sliderTwo.isInitialized() && p.getPopulation() != 0) {
```

Comment:
Is it possible that one is initialized and one isn't? That would cause issues.

Reply:
Nah, it's not possible. But I believe this was code you wrote, so I just modified it.

Reply:
This needs a lil reworking